packages/public/server/src/module/Ui/templates/entity/page/grouped-text-exercise.twig

```diff
@@ -23,11 +23,12 @@
 {% set parent = entity.getParents('link').first() %}
 {% do headTitle(title ~ ' - ' ~ entity().findSubject(parent) ~ ' ' ~ (entity.getType().getName() | trans)) %}
 {% do normalize().headMeta(entity) %}
-{{ pageHeader(normalize().toTitle(entity))
-    .setSubtitle(entity.getType().getName() | trans)
-    .setBacklink(normalize().toUrl(entity.getParents('link').first()))
-    .render(false) }}
-
+{% if not layout().usingAPI %}
+    {{ pageHeader(normalize().toTitle(entity))
```

Comment:
Maybe: remove page headers completely (since ID isn't a good title) and rather only show the type in Breadcrumbs (or don't show it at all)